### PR TITLE
feat: Add mapyde recipe

### PR DESCRIPTION
**Pull request by kratsg**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mhance** [comment]
I approve being listed as a maintainer (I am a maintainer).

**Comment by kratsg** [comment]
@chrisburr  - any suggestions here?

**Comment by BastianZim** [comment]
> In this case, we only need to add some packages for some python versions (but not all) because of inclusion in stdlib.

Conda doesn't have this, instead we select these packages only for specific versions: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors

**Comment by kratsg** [comment]
@chrisburr can you look again?

File: recipes/mapyde/meta.yaml
Comment (carterbox) [inline recipes/mapyde/meta.yaml:38]:
```suggestion
test:
  requires:
    - pip
  imports:
    - mapyde
  commands:
    - mapyde --version
    - pip check
```

File: recipes/mapyde/meta.yaml
Comment (carterbox) [inline recipes/mapyde/meta.yaml:50]:
```suggestion
  description: >
```

**Review by carterbox — APPROVED** [review]
LGTM when pip check passes

**Pull request merged**
